modules/ROOT/pages/update-acm.adoc

```diff
@@ -26,6 +26,7 @@ Attempting to upgrade the package without this feature enabled causes the proces
 . Verify the version number and name of the release.
 . If you are upgrading from a package version before 1.26, follow the <<pre-126>> and then continue with these instructions.
 . If you are upgrading from a package version before 1.74, follow the <<pre-174>> and then continue with these instructions.
+. If you are upgrading from a package version before 1.77, follow the <<pre-177>> and then continue with these instructions.
```

Comment:
I always use earlier/later than, but I don't see anything forbidding this in the SG.